core/spec/lib/factory_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe FactoryBot do
```

Comment:
I don't know if `FactoryBot` is the correct module to describe here. You could just use a 'Factories' string, and I think that would make more sense.

Reply:
That makes sense, thanks!